solveODE.m

```
@@ -52,8 +52,27 @@
     % Set the starting value and signature for the next stage
     solveODE_prepareNextStage(datahandle);
 
+    % chattering/inconsistent switching? --> filippov regime
+    chattering = solveODE_recognizeFilippovSwitching(datahandle);
+    if chattering
+        solveODE_setFilippovRHS(datahandle);
+        % extend solution from t2 until end of filippov regime
+        extendODE_filippov_regime(datahandle);
+        if checkForSwitchingIndices(datahandle)
+            warning('Switching during or right after Filippov mode.');
```

Add identifier to this warning, so it can be turned off.
Also, is a warning enough here? Maybe we should throw an error if we don't have proper handling for this case in place yet.